src/site/pages/accessibility-guide.md

```diff
@@ -0,0 +1,86 @@
+
+Since accessibility is best achieved when it's considered at the beginning, we ask designers and developers who contribute or consume patternfly to understand how accessibility needs can be met. The following is a guide with some techniques on how to design, develop, and test UI to ensure that everyone has a good user experience.
+
+- [Understanding Users’ Needs](#understanding-users-needs)
+- [Checklists](#checklists)
+  - [Design and Development Guidelines](#design-and-development-guidelines)
+      - [What PatternFly Designers and Developers Should Address](#what-patternfly-designers-and-developers-should-address)
+      - [What Product Developers and Designers Should Address](#what-product-developers-and-designers-should-address)
+  - [Testing](#testing)
+  - [Screen Reader Support](#screen-reader-support)
+- [Methods](#methods)
+
+## Understanding Users’ Needs
+
+As with any design, it’s important to understand the user you’re designing for. To develop an inclusive design, it’s important to understand the different needs of [different users](https://a11yproject.com/posts/myth-accessibility-is-blind-people/), and consider the assistive tools and methods they use. The following are some of the ways we can help address these different needs.
+
+### No Vision
+
+Users with no vision rely on screen readers to access web sites and applications. Often, screen reader users will navigate a page by browsing specific elements on the page, like headers, links, or form elements. Using semantic elements and ensuring that these elements are still meaningful when pulled out of context is critical for the usability of the site.
+
+### Low Vision
+
+Users with low vision can have different needs depending on their eye condition. Needs can range from not relying on color to communicate information, color palettes that provide sufficient contrast, and a layout that is responsive when the user zooms in to magnify the contents of the screen.
+
+### Motor
+
+Users with poor motor control can use a range of devices to access contents. If these users are using only a keyboard, then they rely on all elements being keyboard accessible, with elements that have focus being highly visible. If these users are using a mouse or touch, then target areas should be large enough to be easy to hit.
+
+### Cognitive
+
+Users who have difficulty processing information benefit from contents that are well-written. They should be clear and concise, and structured so that the contents are easy to consume by scanning the page. Avoid long paragraphs.
+
+Some users fall into just one category above, but it’s possible for a user to fall into more than one, or use tools and devices designed for a user group they don’t fall into. One of the greatest benefits of inclusive design practices is that methods that are designed for a specific user group will often provide benefits to all users.
+
+
+## Checklists
```

Comment:
We might try something like this --
Get rid of the Checklists title and make the sections as follows:
-Design and Development Guidelines 
-Accessibility Checklist for PatternFly Designers and Developers
-Accessibility Checklist for All Product Designers and Developers

Reply:
I've definitely struggled with how to label this second main section. I'd like to think about it more before making any changes now.